docker/fv3net/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM jupyter/base-notebook as build
 
 # Manually increment this string to invalidate the cache
-ENV MANUAL_CACHE_STRING 1
+ENV MANUAL_CACHE_STRING 2
```

Comment:
build was failing for a reason that seemed unrelated to this PR, and this did resolve it